.github/workflows/php-cs-fixer.yml

```
@@ -22,6 +22,19 @@ jobs:
           php-version: 7.4
           coverage: none
 
+      - name: Determine composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=directory::$(composer config cache-dir)"
+
+      - name: Cache dependencies installed with composer
+        uses: actions/cache@v2.1.6
```

IMHO I don't think it's worth maintaining the version of actions/cache and I suggest to go with `@2`:
```suggestion
        uses: actions/cache@v2
```
PS: this would also match `phpunit.yml` which uses the same